proxy/src/main/java/com/wavefront/agent/ProxyConfig.java

```diff
@@ -1309,6 +1309,22 @@ public class ProxyConfig extends Configuration {
               + "`service`. Default: none")
   String customServiceTags = "";
 
+  @Parameter(
+      names = {"--customExceptionTags"},
+      description =
+          "Comma separated list of log tag "
+              + "keys that should be treated as the exception in Wavefront in the absence of a "
+              + "tag named `exception`. Default: exception, error_name")
+  String customExceptionTags = "";
+
+  @Parameter(
+      names = {"--customLevelTags"},
+      description =
+          "Comma separated list of log tag "
+              + "keys that should be treated as the log level in Wavefront in the absence of a "
+              + "tag named `level`. Default: level, log_level")
+  String customLevelTags = "";
+
```

Comment:
Theres a section later in this file around line 2200 that adds logic for passing in a config file. Can you add these 2 variables there as well.

Reply:
Do i need to add logic for level in **verifyAndInit()** method ?

Reply:
Thanks @xuranchen , That was missed while manually merging code to fork. I have updated changes.

Reply:
I'm talking about this function
https://github.com/wavefrontHQ/wavefront-proxy/blob/a23233f527a5bfde9072edc8906186ba30cce786/proxy/src/main/java/com/wavefront/agent/ProxyConfig.java#L2269

Can you add the 2 variables here.